ai-server/main.py

```
@@ -69,8 +74,7 @@
 
 @app.exception_handler(RequestValidationError)
 async def validation_exception_handler(request: Request, exc: RequestValidationError):
```

Ruff (0.15.12)

[warning] Unused function argument: `request`

(ARG001)